example/kokkos_pi/kokkos_pi.cpp

```diff
@@ -123,9 +123,8 @@ static double sumArray(MeshBlock *firstBlock, const int &n_block) {
   double theSum = 0.0;
   // reduce the sum on the device
   // I'm pretty sure I can do this better, but not worried about performance for this
-  MeshBlock *pmb = firstBlock;
-  while (pmb) {
-    auto &base = pmb->real_containers.Get();
+  for (auto &mb : blocks) {
+    auto &base = mb.real_containers.Get();
     auto inOrOut = base->PackVariables({Metadata::Independent});
```

Comment:
Optional: This is a bit of rabbit hole sense most of these containers do not have const access methods, but really all these should be const because you are not really changing anything internally.

Reply:
That's worth doing, but should probably be a separate issue.

Reply:
Issue created.